notebooks/Shuvomoy - Benders decomposition.html

```diff
@@ -1206,9 +1206,9 @@ <h3 id="How-to-add-lazy-constraints-in-Julia?">How to add lazy constraints in Ju
 
 <div class="output_area"><div class="prompt"></div>
 <div class="output_subarea output_stream output_stdout output_text">
-<pre>*     0+    0                           -4.0000       -3.8333        0    4.17%
-      0     1       -3.8333     1       -4.0000       -3.8333        0    4.17%
-Elapsed time = 0.83 sec. (0.04 ticks, tree = 0.00 MB, solutions = 1)
+<pre>*     0+    0                           -4.0000       -3.8333        0    4.17%
+      0     1       -3.8333     1       -4.0000       -3.8333        0    4.17%
+Elapsed time = 0.83 sec. (0.04 ticks, tree = 0.00 MB, solutions = 1)
```

Comment:
what is the difference here?

Reply:
There doesn't seem to be any difference when we see it here but the git diff on my laptop shows that there were `^M` carriage returns at the end of each of these lines. My guess is that the text editor automatically removed them since they are only used on DOS/Windows systems. I hope that this isn't an issue?
![benders](https://user-images.githubusercontent.com/22664964/53695728-2c353380-3de5-11e9-92f9-e13075384be2.png)